### ISSUE TITLE
Onboarding redirect

### ISSUE THREAD

**Issue by mikehelmick** [google/exposure-notifications-verification-server#1634]
### TL;DR

In the ENX redirect service, implement a top level redirect that prompts for onboarding.

For iOS this results in a redirect to the new ens://onboard spec #1610 and #1632

For Android, redirect to `market://search?q=exposure%20notifications`

### Design

**Proposal**

1. Accept requests at the top level ENX redirect domain (no parameters). i.e. `https://en.express`
    - iOS redirect to `ens://onboarding` and Android redirect to `market://search?q=exposure%20notifications`
    - this will require adding a new domain to the load balancer cert - and a new piece of config for the redirect service.
    - This must be done without causing an SSL outage.

2. Accept top level hits on the registered subdomains, but do the same onboarding flow.
    - iOS, redirect to `ens://onboarding`
    - Android - in the case the region is known, so if there is an associated mobile app redirect to that app in the play store, if it isn't known, use the market link above.
